### PR TITLE
Remove flex from message box

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
Reverted flex style from message box.

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on the final implementation.
- [x] Documentation provided. (Add links if there are any) - N/A
- [x] Unit tests provided. (Add links if there are any) - N/A
- [x] Integration tests provided. (Add links if there are any) - N/A

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [x] Ran FindSecurityBugs plugin and verified report? - N/A
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by nilasini** [comment]
Please don't merge now

**Review by AyoobAboosalih — APPROVED** [review]

**Comment by nilasini** [comment]
You can merge now.

**Comment by DonOmalVindula** [comment]
> You can merge now.

Ack.. will merge after the CI passes

**Review by nilasini — APPROVED** [review]

**Pull request merged**
